extensions-core/stats/src/main/java/io/druid/query/aggregation/teststats/PvaluefromZscorePostAggregator.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.aggregation.teststats;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Sets;
+import io.druid.query.aggregation.AggregatorFactory;
+import io.druid.query.aggregation.PostAggregator;
+import io.druid.query.aggregation.post.PostAggregatorIds;
+import io.druid.query.cache.CacheKeyBuilder;
+import org.apache.commons.math3.distribution.NormalDistribution;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by chunchen on 4/5/17.
+ */
+@JsonTypeName("pvalue2tailedztest")
+public class PvaluefromZscorePostAggregator implements PostAggregator {
+  private final String name;
+  private final PostAggregator field;
+
+  @JsonCreator
+  public PvaluefromZscorePostAggregator(
+      @JsonProperty("name") String name,
+      @JsonProperty("field") PostAggregator field
+  ) {
+    Preconditions.checkNotNull(name, "Must have a valid, non-null post-aggregator");
+    this.name = name;
+    this.field = field;
+  }
+
+  @Override
+  public Set<String> getDependentFields() {
+
+    Set<String> dependentFields = Sets.newHashSet();
+
+    dependentFields.addAll(field.getDependentFields());
+
+    return dependentFields;
+  }
+
+  @Override
+  public Comparator getComparator() {
+    throw new UnsupportedOperationException();
```

Comment:
This means the post-aggregator cannot be sorted on. Would it make sense to use a comparator that understands Doubles, since it seems like `compute` will return a Double? Or would it be useless to ever sort on this post-aggregator?